omas/omas_physics.py

```diff
@@ -203,6 +203,47 @@ def core_profiles_zeff(ods, update=True, use_electrons_density=False):
                 prof1d_z['zeff'] = Z2n/Zn
     return ods_z
 
+
+def transform_current(rho, JtoR=None, JparB=None,
```

Comment:
Maybe:

```python
def transform_current(equilibrium, rho, JtoR=None, JparB=None, includes_bootstrap=False):
```

or else, I think you should handle better the case where the user does not set `equilibrium`